ultra/ultra/utils/episode.py

```diff
@@ -358,12 +359,20 @@ def episodes(n, etag=None, log_dir=None):
                     )
                     for agent_id, agent_info in e.info[e.active_tag].items()
                 ]
+                agent_goal_completion_strings = [
+                    "{}: {}".format(
+                        agent_id,
+                        int(agent_info.data["reached_goal"]),
```

Comment:
It might make more sense to cast to a `bool` here? Let me know your thoughts.
```suggestion
                        bool(agent_info.data["reached_goal"]),
```